volatility3/framework/symbols/linux/extensions/network.py

```diff
@@ -247,17 +265,34 @@ def get_queue_length(self) -> int:
 
 class in_device(objects.StructType):
     def get_addresses(
-        self,
+        self, max_devices=128
```

Comment:
Hmmmm, technically we should bump the minor version so people know when this parameter was introduced, but I'm willing to let it slide in this case.  It's unlikely anyone trying to use it will be having their plugin used with an older version of the framework...